apiserver/facades/agent/uniter/uniter.go

```diff
@@ -984,6 +984,77 @@ func (u *UniterAPI) WatchActionNotifications(args params.Entities) (params.Strin
 	return common.WatchActionNotifications(args, canAccess, watchOne), nil
 }
 
+// WatchActionNotifications returns a StringsWatcher for observing the status of an Upgrade Series
+func (u *UniterAPI) WatchUpgradeSeriesNotifications(args params.Entities) (params.NotifyWatchResults, error) {
```

Comment:
I'm wondering if we can put these into an api shared between the uniter and the new upgrade series worker?  Both need to WatchUpgradeSeriesNotifications(), UpgradeSeriesStatus() etc.  Entities deal in tags, so we can get different data if necessary for machines and units.

Reply:
This code is being moved into a new API as per the above, therefor there is no reason to bump version here.

Reply:
@ExternalReality I'm not sure what you are meaning "as per the above". If we are adding methods to the uniter API, then we need to bump the facade version, even if they are brought in through a common implementation.

Reply:
@howbazaar, the above comment by @hmlanigan. The code is being moved out of the UniterAPI entirely in the next PR.

Reply:
Generally we have a facade per-worker, and while they might both embed a common.Methods, they would be separate grouping of methods.